iaso/odk/validator.py

```diff
@@ -85,7 +89,10 @@ def group_by_lambda(collection, field_name_lambda):
 
 
 def validate_xls_form(xls_file):
-    excel_file = pd.ExcelFile(xls_file, engine="openpyxl")
+    file_extension = os.path.splitext(xls_file.name)[1].lower()
+    engine = "xlrd" if file_extension == ".xls" else "openpyxl"
```

Comment:
Why are we using two engines? Is this a speed issue?

Reply:
because openpyxl = xlsx only 
and xlrd = xls only